feature/experimental/gribi/ate_tests/backup_nhg_action/backup_nhg_action_test.go

```diff
@@ -18,18 +18,37 @@ import (
 )
 
 const (
-	ipv4PrefixLen        = 30
-	ipv6PrefixLen        = 126
-	innerdstPfx          = "203.0.113.55"
-	mask                 = "32"
-	primaryTunnelDstIP   = "203.0.113.1"
-	secondaryTunnelDstIP = "203.0.113.70"
-	secondaryTunnelSrcIP = "198.51.100.222"
-	vrfName              = "VRF-1"
-	NH1ID                = 1
-	NH2ID                = 2
-	NH3ID                = 3
-	innersrcPfx          = "198.51.100.1"
+	ipv4PrefixLen   = 30
+	ipv6PrefixLen   = 126
+	mask            = "32"
+	outerDstIP1     = "198.51.100.1"
+	outerSrcIP1     = "199.51.100.1"
+	outerDstIP2     = "111.111.111.111"
+	outerSrcIP2     = "222.222.222.222"
+	innerDstIP1     = "98.51.100.1"
+	innerSrcIP1     = "99.51.100.1"
+	VIP1            = "100.1.1.1"
+	VIP2            = "101.1.1.1"
+	vrf1            = "VRF-A"
```

Comment:
Shall we make the variable names as `vrfA`, `vrfB` and `vrfC` for the easy of reading?

Reply:
updated